tests/Makefile

```diff
@@ -13,7 +13,8 @@ PROGS = test1 test2 test3 test4 test5 test6 test7 test8 test9   \
         test58 test59 test60 test61 test62 test63 test64 test65 \
         test66 test67 test68 test69 test70 test71 test72 test73 \
         test74 test75 test76 test77 test78 test79 test80 test81 \
-        test82 test83 test84 test85 test86 test87 test88 test89
+        test82 test83 test84 test85 test86 test87 test88 test89 \
+				test90 test91 test92
```

Comment:
This merge-conflicts with 13723b1c0d16e2a80e0f6bf0175aaadcd6a6c8f5; please `git rebase -i upstream/master` and fix the conflict. (Also, looks like some tabs-for-spaces issues crept in here; please fix that too.)  To get an `upstream` remote into your local copy of the repo, you can add these lines to your `uthash/.git/config`:

    [remote "upstream"]
        url = git@github.com:troydhanson/uthash
        fetch = +refs/heads/*:refs/remotes/upstream/*

As far as I'm concerned, these three tests could/should be done in a single `test91.c` file, one after the other.